net.sonic_pi.SonicPi.json

```diff
@@ -185,17 +199,7 @@
                 "/app/server/ruby/vendor/ffi-*"
             ],
             "post-install": [
-                "install -Dm755 -t $FLATPAK_DEST/app/build/gui/qt app/build/gui/qt/sonic-pi",
-                "install -Dm755 -t $FLATPAK_DEST/bin bin/sonic-pi",
-                "sed -e /^eval/d -i $FLATPAK_DEST/bin/sonic-pi",
-                "mkdir -p $FLATPAK_DEST/app/gui/qt/",
-                "cp -R app/gui/qt/theme $FLATPAK_DEST/app/gui/qt/theme",
-                "cp -R etc $FLATPAK_DEST",
-                "install -d $FLATPAK_DEST/app/server/ruby",
-                "cp -R app/server/ruby/*.rb app/server/ruby/bin app/server/ruby/lib $FLATPAK_DEST/app/server/ruby/",
-                "cp -R app/server/ruby/vendor $FLATPAK_DEST/app/server/ruby",
-                "cp -R app/server/native $FLATPAK_DEST/app/server",
-                "install -Dm644 -t $FLATPAK_DEST/app/server/erlang/sonic_pi_server/ebin app/server/erlang/sonic_pi_server/ebin/*.beam",
+                "cd app/build/linux_dist && cp -av * $FLATPAK_DEST",
```

Comment:
There is a new script that prepares a new release with all the needed files in the right place, so instead of manually going one by one like before (some of the stuff has changed so it didn't work _as is_), I thought it's just better run that script (in the build section) and here simply copy the whole result into the tree.

The only ones I left unchanged are the flatpak specific files (icons, desktop file, etc.).